package/upgrade/upgrade_manifests.sh

```diff
@@ -124,6 +134,7 @@ wait_kubevirt()
       fi
     fi
 
+    echo "Libvirt current version: $current_target_version, target version: $version"
```

Comment:
I think it's `KubeVirt`?
```suggestion
    echo "KubeVirt current version: $current_target_version, target version: $version"
```

Reply:
That's embarrassing, thanks for spotting this.